addons/mail/static/src/models/follower/follower.js

```diff
@@ -158,6 +158,10 @@ function factory(dependencies) {
                     args: [[this.followedThread.id]],
                     kwargs,
                 }));
+                this.env.services['notification'].notify({
+                    type: 'success',
+                    message: this.env._t('The subscription preferences were successfully applied.'),
+                });
```

Comment:
I would move this inside the `this.async` param function. The goal of the instruction is to prevent async code from doing operations on `this` record after it has been deleted. However in this case the notification should be tied to the RPC itself, because even if the follower record is deleted we would want to display the notification because the server received the update.

Reply:
@seb-odoo if the user will get removed from the follower's list then the "Message" should be the different one. Right?
it is not same like "The subscription preferences were successfully applied."

Reply:
Not being follower is also a subscription preference, so that would be fine for me.
You can ask PO if we want a different message, I'm not against it, but even then the problem about `async` is the same. If you put the code after it the follower will be deleted and there will be no message at all.

Reply:
@seb-odoo .. as per the PO's suggestion, there is no need to display the notification on removal.

Reply:
Ok thank you for the follow up.